source/AI.cpp

```diff
@@ -1348,6 +1354,17 @@ bool AI::FollowOrders(Ship &ship, Command &command) const
 		else
 			command.SetTurn(TurnToward(ship, TargetAim(ship)));
 	}
+	else if(type == Orders::COLLECT){
+		if(DoHarvesting(ship, command))
+		{
+			ship.SetCommands(command);
+			ship.SetCommands(firingCommands);
+			return true;
+		} else {
```

Comment:
```suggestion
		}
		else
```

Single statements do not include curly braces in this project style.  Example,
```cpp
	else
		CircleAround(ship, command, *target);
```